packages/components/src/theme/ComposerTheme/Components/Skeleton.ts

```diff
@@ -0,0 +1,12 @@
+import { ComponentStyleConfig, cssVar } from '@chakra-ui/react';
+
+export const Skeleton: ComponentStyleConfig = {
+  baseStyle: ({ theme }) => {
+    const $startColor = cssVar('skeleton-start-color');
+    const $endColor = cssVar('skeleton-end-color');
+    return {
+      [$startColor.variable]: theme.colors['background-skeleton-start'],
+      [$endColor.variable]: theme.colors['background-skeleton-end'],
```

Comment:
do we have to reference the `theme`?, cannot we just use the token?

Reply:
Chakra doesn't resolve the string token